### ISSUE TITLE
editor placeholder issue

### ISSUE THREAD

**Issue by KonstantinKellermann** [xamarin/Xamarin.Forms#5822]
**Description**

The placeholder of the material editor looks like to be duplicated or something else.
If I tap into the editor the placeholder is going to the top but also there is a placeholder in the textfield. So there are two placholders after that, one on the top and one in the textfield.

I have created an new project just with an entry and an editor to reproduce it and to show the difference between both of the placeholders.
I am using material design. After an update (Xamarin.Forms.Visual.Material) yesterday the bug was there.

I only tested it on android so I can not tell you if the problem is the same on ios.

**Expected Behavior**

The placeholder should looks like the placeholder of the entry.

**Basic Information**

- Nuget Packages:
     - Xamarin.Forms: 3.6.0.293080
     - Xamarin.Forms.Visual.Material: 3.6.0.293080

     - Xamarin.Android.Support.v7.MediaRouter: 28.0.0.1
     - Xamarin.Android.Support.v7.CardView: 28.0.0.1
     - Xamarin.Android.Support.v7.AppCompat: 28.0.0.1
     - Xamarin.Android.Support.v4: 28.0.0.1
     - Xamarin.Android.Support.Design: 28.0.0.1
     - NETStandard.Library: 2.0.3

- Affected Devices:
     - Samsung Galaxy S5 Mini: Android 6.0 - API 23
     - Samsung Galaxy S7: Android 8.0 - API 26
     - Nexus 5X: Android 8.1.0

**Screenshots**

![Screenshot1](https://user-images.githubusercontent.com/23655725/55622734-517cdf00-57a1-11e9-97a4-470b8004adaa.PNG)

![Screenshot2](https://user-images.githubusercontent.com/23655725/55622741-53df3900-57a1-11e9-893f-bd54d148eb29.PNG)

**Code Example**

https://github.com/KonstantinKellermann/EditorTest

**Comment by gaocan1992** [comment]
Currently I do this fix:

```
public class CustomEditorRenderer : MaterialEditorRenderer {
        public CustomEditorRenderer(Context context) : base(context) {
        }

        protected override void OnElementChanged(ElementChangedEventArgs<Editor> e) {
            base.OnElementChanged(e);
            if (e.NewElement != null && Control != null) {
                  Control.EditText.SetHintTextColor(global::Android.Graphics.Color.Transparent);
            }
        }

}
```

https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/custom-renderer/

**Comment by samhouts** [comment]
closed by #5823
